gluten-ut/spark32/src/test/scala/io/glutenproject/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -239,5 +239,25 @@ class ClickHouseTestSettings extends BackendTestSettings {
     // This test will re-run in GlutenExchangeSuite with shuffle partitions > 1
     .exclude("Exchange reuse across the whole plan")
   enableSuite[GlutenReuseExchangeAndSubquerySuite]
+
+  enableSuite[GlutenDataFrameWindowFramesSuite]
+    .exclude(
+      "rows between should accept int/long values as boundary",
+      "reverse preceding/following range between with aggregation",
+      "SPARK-24033: Analysis Failure of OffsetWindowFunction"
+    )
+  enableSuite[GlutenDataFrameWindowFunctionsSuite]
+    .exclude(
+      "reuse window partitionBy",
+      "reuse window orderBy",
+      "collect_list in ascending ordered window",
+      "collect_list in descending ordered window",
+      "collect_set in window",
```

Comment:
collect_* failed ?

Reply:
we will fixe

Reply:
first_value, last_value, collect_* , these functions doesn't support now.